src/main/java/org/jenkinsci/test/acceptance/junit/Wait.java

```diff
@@ -101,9 +101,9 @@ public Wait(Subject input, ElasticTime time) {
 
     /**
      * Create wait with configurable timer.
-     *
+     * <p>
      * This is useful for timeout waiting wall-clock time to pass.
-     * @see {@link Wait<Subject>(Subject, ElasticTime)}
+     * @see Wait<Subject>
```

Comment:
this seems like an incorrect change - it is pointing to the class doc and not the constructor?

if I have got the syntax right it should be (but this was done without IDE assistance)
```suggestion
     * @see #Wait(Subject, ElasticTime)
```